taf/exceptions.py

```diff
@@ -39,6 +43,10 @@ def __init__(self, message):
         super().__init__("root", message)
 
 
+class KeystoreError(TAFError):
```

Comment:
Does it make sense to subclass `YubikeyError`?

Reply:
That seems confusing to me, but it would make sense to create a common ancestor class. Something like `KeyError`, sublassed by `KeystoreError` and `YubikeyError`?

Reply:
I guess we can live it like this then

Reply:
If at some point we need to catch both exceptions, we should consider creating a superclass. At the moment, that does not occur anywhere